Modules/Chatroom/classes/class.ilChatroom.php

```diff
@@ -104,7 +104,7 @@ protected static function checkPermissions($usrId, $refId, array $permissions)
 		{
 			if($pub_ref_id == $refId)
 			{
-				$hasAccess = $DIC->rbac()->system()->checkAccessOfUser($usrId, $permission, $refId);
+				$hasAccess = $DIC->rbac()->system()->checkAccess($permission, $refId);
```

Comment:
I'm not sure about this. The `$usrId` is passed as an argument, why changing the method call here?